examples/nextjs/package.json

```diff
@@ -9,9 +9,9 @@
     "lint": "next lint"
   },
   "dependencies": {
-    "@pcd/eddsa-ticket-pcd": "^0.4.0",
-    "@pcd/passport-interface": "^0.9.0",
-    "@pcd/zk-eddsa-event-ticket-pcd": "^0.3.0",
+    "@pcd/eddsa-ticket-pcd": ">=0.4.0",
+    "@pcd/passport-interface": ">=0.9.0",
+    "@pcd/zk-eddsa-event-ticket-pcd": ">=0.3.0",
```

Comment:
why did you change `^` to `>=` - is this the correct thing to do?

Reply:
Hmm. In retrospect it probably isn't. I ought to just keep these version numbers in sync with the latest ones.